README.md

```diff
@@ -27,36 +28,56 @@ module "build" {
     aws_account_id      = "xxxxxxxxxx"
     image_repo_name     = "ecr-repo-name"
     image_tag           = "latest"
+
+    # Optional extra environment variables
+    environment_variables = [{
+        name  = "JENKINS_URL"
+        value = "https://jenkins.example.com"
+      },
+      {
+        name  = "COMPANY_NAME"
+        value = "Amazon"
+      },
+      {
+        name = "TIME_ZONE"
+        value = "Pacific/Auckland"
+
+      }]
 }
```

Comment:
@Jamie-BitFlight 
This is a nice solution!
Was it tested (`terraform plan/apply`)?

Reply:
i'm currently using it on a project. It plans and applies.

I think that codebuild also has a bug in the hashing algorithm for 'source.auth'
https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_codebuild_project.go
(line 195)

because I notice that between applies I'm getting a `* module.codebuild.aws_codebuild_project.default: aws_codebuild_project.default: diffs didn't match during apply. This is a bug with Terraform and should be reported as a GitHub Issue.`
error.

This has been happening long before i modified the module though.
As per this badly filled out issue report :-P https://github.com/terraform-providers/terraform-provider-aws/issues/4181